py/TurboSHAKE.py

```diff
@@ -86,4 +86,4 @@ def TurboSHAKE128(M, D, outputByteLen):
     return TurboSHAKE(256, M, D, outputByteLen)
 
 def TurboSHAKE256(M, D, outputByteLen):
-    return TurboSHAKE(512, M, D, outputByteLen)
\ No newline at end of file
+    return TurboSHAKE(512, M, D, outputByteLen)
```

Comment:
My editor prefers a newline. Let me know if you'd like me to revert.

Reply:
Rather than editor-specific settings or personal preferences, may I suggest using a formatter such as Black?

Reply:
Done!